Marvel/Events/LoCG/[2016-2017] Civil War II (Marvel Comics)(LoCG).cbl

```diff
@@ -42,14 +42,14 @@
 <Book Series="Ms. Marvel" Number="7" Volume="2015" Year="2016">
 <Database Name="cv" Series="86114" Issue="531894" />
 </Book>
-<Book Series="Civil War II: Ulysses" Number="1" Volume="2016" Year="2016">
-<Database Name="cv" Series="93373" Issue="546053" />
+<Book Series="Civil War II: Ulysses Infinite Comic" Number="1" Volume="2016" Year="2016">
+<Database Name="cv" Series="91316" Issue="535948" />
```

Comment:
Issue ID should be 535498

https://comicvine.gamespot.com/civil-war-ii-ulysses-infinite-comic-1-tower-of-wis/4000-535498/